apps/daimo-mobile/src/App.tsx

```diff
@@ -7,6 +7,8 @@ import { RpcProvider } from "./logic/trpc";
 import { useSyncChain } from "./sync/sync";
 import { HomeStackNav } from "./view/HomeStack";
 import { useInitNavLinks } from "./view/shared/nav";
+import { useInitNFC } from "./logic/nfcListen";
+
 
 export default function App() {
```

Comment:
you can fix all the lint via VSCode > click Install Recommended in the extensions tab > save the file (should auto-format) > Cmd+Shift+P Organize Imports (fixes imports)